deadlock_examples/Account.rb

```diff
@@ -0,0 +1,29 @@
+class Account 
+  include Celluloid
+
+  def initialize
+    @mutex = Mutex.new
+    @balance = 500
+  end
+
+  def withdraw(amount)
+   @balance = @balance - amount;
+ end
+
+ def deposit(amount)
+   @balance = @balance + amount;
+ end
+
+ def transfer(from, to, amount)
+  @mutex.lock
+  from.withdraw(amount)
+  to.deposit(amount)
+  @mutex.unlock
+  end
+end
+
+account1 = Account.new
```

Comment:
You should always instantiate the classes you need in the tests themselves, otherwise these will get run when the examples are loaded -- which will be before the tests happen, and it will disrupt other tests.